src/klein/_app.py

```diff
@@ -184,7 +186,9 @@ class Klein:
     def __init__(self) -> None:
         self._url_map = Map()
         self._endpoints: Dict[str, KleinRouteHandler] = {}
-        self._error_handlers: List[KleinErrorHandler] = []
+        self._error_handlers: List[
+            Tuple[List[Type[Exception]], KleinErrorHandler]
```

Comment:
Minor comment.
Do we need to have a `List` here? Can we have a `Sequence` ?

Reply:
It's an internal-to-the-class non-public data structure, and it's a `List`, because we `append` to it.

Reply:
True. Thanks :)